main.tf

```diff
@@ -146,7 +146,7 @@ resource "aws_iam_policy_attachment" "main" {
 resource "aws_cloudwatch_log_group" "main" {
   count             = "${var.enable_cloudtrail ? 1 : 0}"
   name              = "${var.cloudtrail_cloudwatch_logs_group}"
-  retention_in_days = "${var.expiration}"
```

Comment:
Since this is already a breaking change, I think it would make sense to update the `expiration` variable to something specific to s3. How about `s3_log_bucket_retention`?